src/utils/style-formatter.js

```diff
@@ -29,8 +29,13 @@ const getColor = ({ useColorExpression, colorExpression, color }, defaultColor,
   return resolvedColor === 'none' ? defaultColor : resolvedColor;
 };
 
-const setFontsize = (textElement, newFontsize, fontFamily) => {
-  textElement.setAttribute('style', `white-space: nowrap; font-size: ${newFontsize}px; font-family: ${fontFamily};`);
+const setFontsize = (textElement, newFontsize, fontFamily, addFlex, alignment) => {
+  textElement.setAttribute(
+    'style',
+    `${
+      addFlex ? `display: flex; align-items: center; justify-content: ${alignment};` : ''
+    } font-size: ${newFontsize}px; font-family: ${fontFamily};`
```

Comment:
Setting `font-family` every time the `setFontSize` is called doesn't really make sense. Maybe set it last in `setFontSizeAndFamily`. Use `textElement.style.fontSize()` to not override other styling.